care/facility/tests/test_ambulance_api.py

```diff
@@ -0,0 +1,298 @@
+from typing import Any
+
+from rest_framework import status
+
+from care.facility.models import Ambulance, AmbulanceDriver, District
+from care.utils.tests.test_base import TestBase
+from config.tests.helper import mock_equal
+
+
+class TestAmbulance(TestBase):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method 'get_list_representation' is abstract in class 'TestBase' but is not overridden](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5398997)